<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>637</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>10</horstretch>
    <verstretch>10</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Kroket</string>
  </property>
  <property name="windowIcon">
   <iconset resource="kroket.qrc">
    <normaloff>:/resources/gfx/kroket-icon.svg</normaloff>:/resources/gfx/kroket-icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExportImage"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomFit"/>
    <addaction name="actionZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionExpandAll"/>
    <addaction name="actionCollapseAll"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionHelpInput"/>
    <addaction name="separator"/>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionExportImage"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomFit"/>
   <addaction name="actionZoomOut"/>
   <addaction name="separator"/>
   <addaction name="actionExpandAll"/>
   <addaction name="actionCollapseAll"/>
   <addaction name="separator"/>
   <addaction name="actionReset"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/quit.png</normaloff>:/resources/gfx/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/open.png</normaloff>:/resources/gfx/open.png</iconset>
   </property>
   <property name="text">
    <string>Open ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/zoom-in.png</normaloff>:/resources/gfx/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>=</string>
   </property>
  </action>
  <action name="actionZoomFit">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/zoom-fit.png</normaloff>:/resources/gfx/zoom-fit.png</iconset>
   </property>
   <property name="text">
    <string>Fit View</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/zoom-out.png</normaloff>:/resources/gfx/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionExportImage">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/export-image.png</normaloff>:/resources/gfx/export-image.png</iconset>
   </property>
   <property name="text">
    <string>Export Image</string>
   </property>
  </action>
  <action name="actionExpandAll">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/expand.png</normaloff>:/resources/gfx/expand.png</iconset>
   </property>
   <property name="text">
    <string>Expand All</string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/collapse.png</normaloff>:/resources/gfx/collapse.png</iconset>
   </property>
   <property name="text">
    <string>Collapse All</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/reset.png</normaloff>:/resources/gfx/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/print.png</normaloff>:/resources/gfx/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/undo.png</normaloff>:/resources/gfx/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/redo.png</normaloff>:/resources/gfx/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="kroket.qrc">
     <normaloff>:/resources/gfx/help.png</normaloff>:/resources/gfx/help.png</iconset>
   </property>
   <property name="text">
    <string>Contents</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About Kroket</string>
   </property>
  </action>
  <action name="actionHelpInput">
   <property name="text">
    <string>Kroket Input Format</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="kroket.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
